financements/aides-publiques.md

```diff
@@ -15,7 +15,7 @@ Pour compléter votre apport personnel, vous pouvez demander un prêt d’honneu
 
 ## Avantages
 
-Bien que les montants accordés dans le cadre de prêts d’honneur ou à taux zéro dépassent rarement 50 000 €, l’obtention de ce type de crédit peut faciliter l’obtention de crédits traditionnels par la suite, car ils sont considérés comme des fonds propres, ce qui donne du poids à votre dossier. Un prêt d’honneur de 50 000 € peut par exemple vous aider à décrocher un crédit bancaire de 200 000 €.
+Bien que les montants accordés dans le cadre de prêts d’honneur ou à taux zéro dépassent rarement 50 000 €, l’obtention de ce type de crédit peut faciliter l’obtention de crédits traditionnels par la suite, car ils sont considérés comme des fonds propres, ce qui donne du poids à votre dossier. Un prêt d’honneur de 50 000 € peut par exemple vous aider à décrocher un [crédit bancaire]({{ 'financements/credit-bancaire' | relative_url }}) de 200 000 €.
```

Comment:
We could use this mechanism instead, couldn't we? https://jekyllrb.com/docs/liquid/tags/#link

It would make links sturdier since if a file is renamed, it would fail at buildtime.

What do you think?

Reply:
> We could use this mechanism instead, couldn't we?

That's what we were doing in the first place, actually, but links would broke either on GitHub Pages or locally because the property `baseurl:` wasn't empty at that time—its value was `/growth-guide`, because our site lives in the `growth-guide` repo. 

On PR #8, I wrote:

> - https://guide.memo.bank/growth-guide/assets/css/just-the-docs-default.css is now broken
> - https://guide.memo.bank/assets/css/just-the-docs-default.css works well (note the absence of /growth-guide/)

Of course, if links fail locally, we can always use `bundle exec jekyll serve --baseurl=""` to run the site as if it wasn't located in a subdirectory, but this seemed tedious to me, so I implemented the `relative_url` Liquid filter instead. Now that our site lives at guide.memo.bank, I wonder how links will behave if we remove the relative_url Liquid filter. 🤔

This filter does the following:

> It prepends `baseurl` configuration value to the input creating a relative URL. Recommended for sites placed in a sub-directory e.g., projects hosted on GitHub Pages.

— [Source](https://mademistakes.com/mastering-jekyll/how-to-link/)

Now that our `baseurl` configuration value is empty, I guess we don't need the `relative_url` Liquid filter anymore. Let's switch to links that look like that: `[plain text]({% link garanties/caution-personnelle.md %})`. 👍